Scripts/script-TaniumAskQuestion.yml

```diff
@@ -128,3 +129,4 @@ dependson:
   - tn-result-data
   - tn-result-info
 timeout: 0s
+releaseNotes: Deprecated script
```

Comment:
please write the command that replace this script.
Deprecated script, use `command name` instead.